lib/transferengineclient.cpp

```diff
@@ -230,6 +230,7 @@ int TransferEngineClient::createDownloadEvent(const QString &displayName,
                                               const QUrl &url,
                                               const QString &mimeType,
                                               qlonglong expectedFileSize,
+                                              const bool privateTransfer,
```

Comment:
Minor detail, but I don't see reasons to have const on by-value parameters. The above file size also doesn't have.